modoptions.lua

```diff
@@ -505,6 +505,19 @@ local options = {
         section	= "options",
     },
 
+    {
+        key		= "wreckage_metal_ratio",
+        name	= "Unit Wreck Metal Percentage",
+        desc	= "Percent of unit metal that is left in its wrecks (and heaps)",
+        hidden 	= true,
+        type	= "number",
+        section	= "options",
+        def		= 0.6,
+        min		= 0.2,
+        max		= 0.8,
```

Comment:
There's "0.2 - 0.8" comments elsewhere that will become obsolete if this is changed, perhaps worth leaving some sort of "please look at usage first before changing" remark.